### ISSUE TITLE
Question: The staking UTXO has been spent

### ISSUE THREAD

**Issue by jolestar** [babylonlabs-io/staking-indexer#27]
I checked the indexer record and found some staking UTXO has been spent.

```
13a36cdebcf5a6422ca854fcd036d2c968311ff799eca1452d77e84783233e0b,0,864794,a0d9aa5a96cb12b5ee066bda6a34625ce6efac412749add47c6f065f612159f2,64000,4bf609ba8977d3fbf4dee7f9d993c41f2fa584ccd27b3e4bf04a5376267e13c0,false,1500000
```
The staking output is `0`, but the 0 output has been spent. 
https://mempool.space/tx/13a36cdebcf5a6422ca854fcd036d2c968311ff799eca1452d77e84783233e0b

Is it working as expected?

There are some other cases.

**Comment by gitferry** [comment]
Yep, this can be spent due to early unbonding. The indexer does not care whether the staking output is already spent or not.
